modules/framework/galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -1586,6 +1586,15 @@ paths:
           schema:
             type: string
           example: a6847035-7768-4416-91a6-fe05cf10da85
+        - name: detail
+          in: query
+          description: |
+            Controls the amount of details extracted from a run's test structure and included in responses.
+            Providing "methods" will include the list of methods that were executed as part of a test run.
+            Only "methods" is supported yet.
```

Comment:
Minor - "yet" isn't needed here.

Still might be worth saying whether the value for the `detail` query parameter is case-sensitive here. Someone could try providing `METHODS` but then be faced with an error?